docs/UserGuide.md

```diff
@@ -71,79 +74,136 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Adding an owner: `owner`
+
+Adds an owner to PawPatrol.
+
+Format: `owner n/NAME p/PHONE e/EMAIL a/ADDRESS`
 
-### Adding a person: `add`
+Examples:
+* `owner n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25`
+
+### Adding a pet: `pet`
 
-Adds a person to the address book.
+Adds a pet to PawPatrol.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `pet n/NAME s/SPECIES b/BREED a/AGE sex/SEX [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A pet can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `pet n/Fluffy s/Dog b/Golden Retriever a/7 x/F`
+* `pet n/Megatron s/Cat b/Siamese a/3 x/M t/playful`
+
+### Linking owners and pets: `link`
+
+Links an owner to one or more pets.
+
+Format: `link oOWNER_INDEX t/pPET_INDEX…​`
+
+Examples:
```

Comment:
We could add a picture of the GUI here to show users how it will look once a pet and owner are successfully linked.